docs/yaml-event-classes.rst

```diff
@@ -4,6 +4,11 @@
 Event Classes
 #############
 
+.. note::
```

Comment:
Can you keep this note further down. I don't think it's generally a good thing to jump into notes before even the first paragraph explaining a section.